platform/Env.roc

```diff
@@ -66,7 +66,7 @@ var! = \name ->
 ## ```
 ## # Reads "NUM_THINGS" and decodes into a U16
 ## get_u16_var! : Str => Result U16 [VarNotFound, DecodeErr DecodeError] [Read [Env]]
-## get_u16_var! = \var -> Env.decode! var
+## get_u16_var! = \var -> Env.decode!(var)
```

Comment:
PNC and also snake casing missed in the below `decode!` docs, and the `ARCH`/`OS`.

(can't comment on it since it's out of range of a changed line)